src/db/fakeData.js

```diff
@@ -0,0 +1,147 @@
+          week: ['week1', 'week2', 'week3'],
+          studentScore: [1,2,3,4,5,6,7,8,9,10]
+      },
+      classAttendance: {
+          module: ['Javascript1', 'Javascript2', 'Javascript3', 'Html1', 'Html2', 'Html3', 'React1', 'React2', 'React3'],
+          week: ['week1', 'week2', 'week3'],
+          attended: ['yes', 'no', 'late']
+        },
+      pdSkills: {
+          skill: ["motivation", "communication", "confidence"],
+          skillLevel: ["Excellent", "Very good", "Average", "Below Average", "Poor"]
+      }
+    },
+    {
+        profile:2,
+        studentInfo: {
+              name: "Dennis",
+              location: "Birmingham",
+              email: "dennis123@yahoo.com",
+              phone: "07771112223",
+              slackId: "denis321"
+  
+      },
+      overview: {
+          homeworkPerformance: [1,2,3,4,5,6,7,8,9,10],
+          homeworkMissing: [1,2,3,4,5,6,7,8,9,10],
+          classessAttended: [1,2,3,4,5,6,7,8,9,10],
+          classessLate: [1,2,3,4,5,6,7,8,9,10]
+        },
+      eduHomework: {
+          module: ['Javascript1', 'Javascript2', 'Javascript3', 'Html1', 'Html2', 'Html3', 'React1', 'React2', 'React3'],
+          week: ['week1', 'week2', 'week3'],
+          studentScore: [1,2,3,4,5,6,7,8,9,10]
+      },
+      classAttendance: {
+          module: ['Javascript1', 'Javascript2', 'Javascript3', 'Html1', 'Html2', 'Html3', 'React1', 'React2', 'React3'],
+          week: ['week1', 'week2', 'week3'],
+          attended: ['yes', 'no', 'late']
+        },
+      pdSkills: {
+          skill: ["motivation", "communication", "confidence"],
+          skillLevel: ["Excellent", "Very good", "Average", "Below Average", "Poor"]
+      }
+    },
+    {
+        profile:3,
+        studentInfo: {
+              name: "lola",
+              location: "Birmingham",
+              email: "lola123@yahoo.com",
+              phone: "07771112223",
+              slackId: "lola321"
+  
+      },
+      overview: {
+          homeworkPerformance: [1,2,3,4,5,6,7,8,9,10],
+          homeworkMissing: [1,2,3,4,5,6,7,8,9,10],
+          classessAttended: [1,2,3,4,5,6,7,8,9,10],
+          classessLate: [1,2,3,4,5,6,7,8,9,10]
+        },
+      eduHomework: {
+          module: ['Javascript1', 'Javascript2', 'Javascript3', 'Html1', 'Html2', 'Html3', 'React1', 'React2', 'React3'],
+          week: ['week1', 'week2', 'week3'],
+          studentScore: [1,2,3,4,5,6,7,8,9,10]
+      },
+      classAttendance: {
+          module: ['Javascript1', 'Javascript2', 'Javascript3', 'Html1', 'Html2', 'Html3', 'React1', 'React2', 'React3'],
+          week: ['week1', 'week2', 'week3'],
+          attended: ['yes', 'no', 'late']
+        },
+      pdSkills: {
+          skill: ["motivation", "communication", "confidence"],
+          skillLevel: ["Excellent", "Very good", "Average", "Below Average", "Poor"]
+      }
+    }
+    
+  ]
+  //route for /student-tracker
+   export const studentProgressTracker = [
```

Comment:
studentProgressTracker should be an array with all the students Profile objects.

Reply:
For all the Student Tracker areas (homework, PD, attendance) would be taking from the same array/file?

Reply:
I have altered the data as requested, Can you please review

Reply:
Great guys, I think this will cover all the data request from the React.